sphinx/domains/python.py

```diff
@@ -674,6 +674,8 @@ def get_signature_prefix(self, sig: str) -> str:
         if 'classmethod' in self.options:
             prefix.append('classmethod')
         if 'property' in self.options:
+            logger.warning(__('Use the directive `py:property` instead of '
+                              'the directive `py:method` for properties.'))
```

Comment:
Hmm... it seems it is better to separate deprecation to another PR to avoid breaking tests.